<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewInstrument</class>
 <widget class="QWidget" name="NewInstrument">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>654</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="adiscope::DetachDragZone" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>654</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mainWidget {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.1, y2:0, stop:0 rgba(0, 0, 0, 40), stop:1 transparent);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="hLayout_top_btn_area">
      <property name="spacing">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="printBtn">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
  width: 80px;
  height: 40px;

  text-align: left;
  font-weight: bold;
  padding-left: 15px;
  padding-right: 15px;
  border-radius: 4px;

  background-color: #4a64ff;
  color: #ffffff;

}

QPushButton:pressed {
  background-color: #2a44df;
}

QPushButton:hover {
	background-color: #4a34ff;
}</string>
        </property>
        <property name="text">
         <string>Print</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/ic printer.svg</normaloff>:/icons/ic printer.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="adiscope::RunSingleWidget" name="runSingleWidget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayout_central_area">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="plot_and_buffPreviewer">
        <item>
         <widget class="QWidget" name="widgetPlotContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="plot_and_scales">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayoutPlot">
               <property name="leftMargin">
                <number>25</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>25</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="scales">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="chn_scales">
                 <property name="spacing">
                  <number>15</number>
                 </property>
                 <property name="leftMargin">
                  <number>10</number>
                 </property>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>15</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vLayout_for_rightMenu">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="adiscope::MenuAnim" name="rightMenu" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QStackedWidget" name="stackedWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>4</number>
             </property>
             <widget class="QWidget" name="channelSettings"/>
             <widget class="QWidget" name="logicSettings">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QCheckBox {
  spacing: 8px;
  background-color: transparent;
  font-size: 14px;
  font-weight: bold;

color: rgba(255, 255, 255, 153);
}

QCheckBox::indicator {
width: 14px;
  height: 14px;
  border: 2px solid rgb(74,100,255);
  border-radius: 4px;
}
QCheckBox::indicator:unchecked { background-color: transparent; }
QCheckBox::indicator:checked { background-color: rgb(74,100,255); }

QPushButton {
width: 40px;
  height: 20px;
  background-color: transparent;
}
QPushButton:pressed { border-image: url(:/icons/setup_btn_checked.svg); }
QPushButton:!pressed { border-image: url(:/icons/setup_btn_unchecked.svg); }
QPushButton:hover:!pressed:!checked { border-image: url(:/icons/setup_btn_hover.svg); }
QPushButton:checked { border-image: url(:/icons/setup_btn_checked.svg); }

QLabel {
font-size: 14px;
  font-weight: bold;

color: rgba(255, 255, 255, 153);
}</string>
              </property>
              <layout class="QVBoxLayout" name="logicSettingsLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>15</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>15</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
             <widget class="QWidget" name="cursorsSettings"/>
             <widget class="QWidget" name="measureSettings"/>
             <widget class="QWidget" name="generalSettings"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayout_bottom_btn_area">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="channelsList_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="adiscope::CustomPushButton" name="btnAddMath">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/menu/add.svg</normaloff>:/menu/add.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="adiscope::CustomPushButton" name="btnGetData">
        <property name="styleSheet">
         <string notr="true">
QPushButton {
  width: 50px;
  height: 40px;

  text-align: left;
  font-weight: bold;
  padding-left: 15px;
  padding-right: 15px;
  border-radius: 4px;

  background-color: #4a64ff;
  color: #ffffff;

}

QPushButton:pressed {
  background-color: #2a44df;
}

QPushButton:hover {
	background-color: #4a34ff;
}</string>
        </property>
        <property name="text">
         <string>GetData</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_21">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
  spacing: 8px;
  background-color: transparent;
  font-size: 14px;
  font-weight: bold;
  
  color: rgba(255, 255, 255, 153);
}

QCheckBox::indicator {
  width: 14px;
  height: 14px;
  border: 2px solid rgb(74,100,255);
  border-radius: 4px;
}
QCheckBox::indicator:unchecked { background-color: transparent; }
QCheckBox::indicator:checked { background-color: rgb(74,100,255); }

QPushButton {
  width: 40px;
  height: 20px;
  background-color: transparent;
}
QPushButton:pressed { border-image: url(:/icons/setup_btn_checked.svg); }
QPushButton:!pressed { border-image: url(:/icons/setup_btn_unchecked.svg); }
QPushButton:hover:!pressed:!checked { border-image: url(:/icons/setup_btn_hover.svg); }
QPushButton:checked { border-image: url(:/icons/setup_btn_checked.svg); }

QLabel {
  font-size: 14px;
  font-weight: bold;
  
  color: rgba(255, 255, 255, 153);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QCheckBox" name="boxCursors">
           <property name="text">
            <string>Cursors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="adiscope::CustomPushButton" name="btnCursors">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_22">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>12</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_23">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>12</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Trigger</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="adiscope::CustomPushButton" name="btnTrigger">
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>adiscope::DetachDragZone</class>
   <extends>QWidget</extends>
   <header>detachdragzone.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>adiscope::MenuAnim</class>
   <extends>QWidget</extends>
   <header>menu_anim.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>adiscope::CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>customPushButton.hpp</header>
  </customwidget>
  <customwidget>
   <class>adiscope::RunSingleWidget</class>
   <extends>QWidget</extends>
   <header>runsinglewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
